src/driver/kvm.rs

```diff
@@ -165,6 +167,7 @@ impl Introspectable for Kvm {
                         old,
                     },
                     KVMiEventType::PauseVCPU => panic!("Unexpected PauseVCPU event. It should have been popped by resume VM. (Did you forget to resume your VM ?)"),
+                    _ => unimplemented!()
```

Comment:
Quick fix so everything compiles neatly. This should be removed anyways as soon as we are starting to merge @arnabcs17b006's PRs